cypress/e2e/buy-shirt.cy.ts

```diff
@@ -0,0 +1,34 @@
+describe("Buy a t-shirt", () => {
+  it("then the t-shirt should be bought", () => {
+    cy.visit("http://automationpractice.com/");
+    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
+    cy.get(
+        "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
+    ).click();
+    cy.get("[style*='display: block;'] .button-container > a").click();
+    cy.get(".cart_navigation span").click();
+
+    cy.get("#email").type("aperdomobo@gmail.com");
+    cy.get("#passwd").type("WorkshopProtractor");
+
+    /* click in a button that has id equals to 'SubmitLogin',
+    which is the Sign In button */
+    cy.get("button[id='SubmitLogin']").click();
+    /* Click in the span label which is inside a button label and, which at the
+    same time is inside in a class named cart_navigation. This is the 'proceed to
+    checkout button */
+    cy.get(".cart_navigation button span").click();
```

Comment:
You could delete span that is specific. just click the button 
Same in line 23 and 28